chirp/drivers/tdh8.py

```diff
@@ -130,14 +130,7 @@
 
 #seekto 0x1A08;
 struct{
-    u8 block8:1,
-       block7:1,
-       block6:1,
-       block5:1,
-       block4:1,
-       block3:1,
-       block2:1,
-       block1:1;
+    u8 flags;
```

Comment:
This doesn't need to be inside a struct anmore, it can just be at the top level. But, there's also a "bit" type which never gets used, but it should be used more. So:
```
bit flags[200];
```

And then you can do something like:
```
memobj.flags[memory.number - 1] = int(mem.empty)
```
And not have to do the masking below.

Neither are critical, but just FYI if you want to clean it up more ;)